CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+
+Available categories include the following:
+
+- Ansible
+- Architecture
+- Automation
+- AWS
+- Azure
+- Chef
+- Cloud Files
+- Cloud Monitoring
+- Cloud-networks
+- Cloud Servers
+- Configuration Management
+- Database
+- Developers
+- DevOps
+- Docker
+- Events
+- General
+- Java
+- Jclouds
+- Jenkins
+- Mailgun
+- Neutron
+- NodeJS
+- OpenStack
+- Oracle
+- Orchestration
+- OSAD
+- Private Cloud
+- Python
+- Salesforce
+- SDK
+- Security
+- SQL Server
+
+If no category fits, use *General*.
+
+If you'd like to use a category that is not in the list, please send an email
+to infodev@rackspace.com. To avoid being flooded with categories, which might apply
+to only one or two blogs, we have automated throttling. However, notify us so
+that we can discuss your ideas for a new category.
+
+##### Excerpt
+
+Include an excerpt marker after your first paragraph or so to separate the
+preview text that appears on the blog index page from the full article. To do
+so, use the following HTML comment:
+
+```
+The excerpt paragraph, which should give teh reader a taste of what's to come.
+
+<!-- more -->
+
+The rest of your article.
+```
+
+The marker comment ``<!-- more -->`` must be on its own line, starting at
+column 1, and separated from content on either side by a single blank line.
+
+##### Images
+
+**To include images in your blog**, place them  within their respective blog
+directory. Within your blog, use the following markup:
+
+```markdown
+![Alt text here]({% asset_path YYYY-MM-DD-title-of-your-post/filename.png %})
+```
+
+**To wrap text around an image on the right**, use code similar to the following:
+
+```
+<img class="blog-post right" src="{% asset_path 2015-06-17-built-an-app-on-openstack-at-qcon-ny-2015/qcon.png %}"/>Last week I went to QCon NY 2015 to be both a student and a teacher in their tutorial track. They follow the standard pattern of having 2 days of tutorials prior to the conference proper. To understand QCon a bit better, here's their mission statement.
+```
+
+##### Writing your post
+
+We recommend that you follow the [Style Guide](https://developer.rackspace.com/docs/style-guide/).
+
+##### Submitting your blog entries
+
+Follow these steps to submit your entry for publication.
+
+1. Submit a PR (pull request) against `master` branch.
+2. Do everything in the PR template checklist.
+3. Once the PR successfully builds, you can preview your entry by clicking on
+   the preview link in the Git GUI.  Sometimes, if you push a new commit to your
+   PR, the changes don't show up in the new staging link.  To force a rebuild,
+   add a blank line to the bottom of your post and commit the change.
+4. When you're satisfied, the InfoDev team will do a quick editorial review
+   of the content and suggest changes, if necessary.
+5. After the content is ready to go, the InfoDev team will publish the blog on
+   the date that you selected.
+
+Thanks again for your interest in contributing!
+
+##### Ideas to post about
```

Comment:
I get that this comes from search terms, but it seems quite outdated and not where we want to go. let's get a new list.

Reply:
Pretty sure this is less than six months old.  How frequently would you like it updated?

Reply:
Note: Heather Fergusson says that an agency collects this info for us, supposedly on a quarterly basis.  However, they are currently off-cadence.  I've asked to be added the list to get access to updates whenever they happen.  If I get that access, I'll update the top search terms as often as we get a new set.

Reply:
@catlook Trending search terms is something that we can ponder internally - but - the goal of this section is to ask for what we *want* to be contributed. I don't think we *want* new articles for many of these dated terms e.g. kibana, micron, java ee, mobile world congress, etc.